### PR TITLE
DO TODO: remove this alias after 2017-05-22

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

File: build/root/BUILD.root [hunk 1]
```diff
@@ -33,13 +33,6 @@ gcs_upload(
     },
 )
 
-# TODO: remove this alias after 2017-05-22
```
Comment (xiangpengzhao) [inline build/root/BUILD.root]: just curious, why this date? Is there any dependency?

Comment (mikedanese) [inline build/root/BUILD.root]: Nope. We just want to enforce deprecation.

Comment (xiangpengzhao) [inline build/root/BUILD.root]: Thanks Mike!

**Comment by ixdy** [comment]
blocked by https://github.com/kubernetes/test-infra/pull/2867

**Comment by ixdy** [comment]
/retest

this LGTM assuming the test-infra PR is merged

**Comment by mikedanese** [comment]
lgtm'ing per ixdy's comment.

**Pull request merged**